### DISCUSSION TITLE
how to get scatter graph with data on x and y axis

### DISCUSSION THREAD

**Discussion by derSchreifritz** [dbuezas/lovelace-plotly-graph-card#80] [Q&A]
Hi,
I would like to have not datetime but sensor data in both x and y axis. How can I achieve this?
Use case is to display daily heatpump cop vs average temperature,

**Comment by dbuezas** [comment]
That's kind of fiddly at the moment, and I'm not sure it would work, but you can try this: (warning, you need to know a bit of JavaScript)
* Add the first entitys and using a lambda, sore the ys and xs of the first one in "window.sensor1". From the lambda return empy xs and ys so it's not plotted
* Add the second one, and in a lambda grab the first sensor data. Align the the xs, and then return {xs:sensor1.ys, ys: ys}

**Reply by derSchreifritz** [reply]
thanks for the reply. I don't know JavaScript or much else for that matter. Will look into it. I managed to combine my 2 sensors into one with template. So that one has both COP, Celsius values with one timestamp I guess. It would be nice in the end to have the new dots bigger than older ones. But that's for later.
